src/components/footer/footer.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+
+import Icon from './icon';
+import facebook from './facebook.svg';
+import github from './github.svg';
+import medium from './medium.svg';
+import twitter from './twitter.svg';
+
+const styles = theme => ({
+	container: {
+		backgroundColor: '#352A3A'
+	},
+	buttonLink: {
+		textDecoration: 'none'
+	},
+	content: {
+		maxWidth: theme.maxWidth,
+		padding: theme.spacing(1, 3, 1, 3),
+		margin: 'auto',
+		display: 'flex',
+		flexDirection: 'row',
+		justifyContent: 'space-between',
+		alignItems: 'center',
+		fontFamily: ['Poppins', 'sans-serif'],
+		fontSize: theme.typography.fontSize,
+		color: '#FFFFFF',
+		textAlign: 'center',
+		[theme.breakpoints.down('xs')]: {
+			flexDirection: 'column'
+		}
```

Comment:
Now that I looked at it, maybe instead we can use M-UI's `Grid` component, which internally still uses flexbox, to control how the 3 children components align. But the M-UI `Grid` let us control how the component float (column or row) depending on screen size, instead of having us to change it in CSS. It is great that you can implement this with CSS. That means you have a great grasp of flexbox. I also want you to learn the M-UI approach to this. 

Try to look at the M-UI grid demo and docs. You can ask me questions during next dev meeting.